src/plugins/vis_type_xy/public/editor/components/common/truncate_labels.test.tsx

```diff
@@ -0,0 +1,51 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0 and the Server Side Public License, v 1; you may not use this file except
+ * in compliance with, at your election, the Elastic License 2.0 or the Server
+ * Side Public License, v 1.
+ */
+
+import React from 'react';
+import { mountWithIntl } from '@kbn/test/jest';
+import { ReactWrapper } from 'enzyme';
+import { TruncateLabelsOption, TruncateLabelsOptionProps } from './truncate_labels';
+import { findTestSubject } from '@elastic/eui/lib/test';
+
+describe('TruncateLabelsOption', function () {
+  let props: TruncateLabelsOptionProps;
+  let component: ReactWrapper<TruncateLabelsOptionProps>;
+
+  beforeAll(() => {
+    props = {
+      disabled: false,
+      value: 20,
+      setValue: jest.fn(),
+    };
+  });
+
+  it('renders an input type number', () => {
+    component = mountWithIntl(<TruncateLabelsOption {...props} />);
+    expect(findTestSubject(component, 'xyLabelTruncateInput').length).toBe(1);
+  });
+
+  it('renders the value on the input number', function () {
```

Comment:
nit: could you please use one style for coding. I see for some places you use `() => {`, for some `function`